utils/collect-updates/collect-updates.js

```diff
@@ -65,6 +65,14 @@ function collectUpdates({ filteredPackages, packageGraph, options, execOpts, log
     return Array.from(candidates);
   }
 
+  if (!project.isIndependent() && options.exact && candidates.size > 1) {
```

Comment:
if you run `lerna publish --exact` on a fixed version project, all packages should bump together regardless whether they have changed or not

Reply:
No, that's incorrect. `--exact` only affects the dependency range used, not the presence/absence of changes.